lib/presentation/home/bloc/home_bloc.dart

```diff
@@ -77,6 +81,11 @@ class HomeBloc extends Bloc<HomeEvent, HomeState> with StreamSubscriptions {
     }
   }
 
+  Future<void> _onTapClearLolPathHomeEvent(TapClearLolPathHomeEvent event, Emitter<HomeState> emit) async {
+    _lcuStore.clear();
```

Comment:
Lets not to directly access lcuStore, i think it's better to create a `_lcu.resetPath` method

Reply:
I was thinking the same